components/brave_vpn/brave_vpn_service_desktop.h

```diff
@@ -124,11 +126,14 @@ class BraveVpnServiceDesktop
   std::unique_ptr<brave_vpn::Hostname> PickBestHostname(
       const std::vector<brave_vpn::Hostname>& hostnames);
 
+  void OnSkusRemoteDisconnect();
```

Comment:
convention is just `OnDisconnect`

Reply:
This one is a bit weird, since it's a VPN class which has methods already for connect/disconnect/etc which are handlers for when the VPN's state changes. I can change to `OnSkusServiceDisconnect` or similar

Reply:
Renamed in https://github.com/brave/brave-core/pull/10847/commits/3af3a81f930c0fcaaf61080214915786f3a070f5